src/packages/core/collection/components/collection-view-bundle.element.ts

```diff
@@ -0,0 +1,95 @@
+import { UMB_COLLECTION_CONTEXT, UmbCollectionContext } from '../collection.context.js';
+import { ManifestCollectionView } from '../../extension-registry/models/collection-view.model.js';
+import { css, html, customElement, state, nothing } from '@umbraco-cms/backoffice/external/lit';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+
+@customElement('umb-collection-view-bundle')
+export class UmbCollectionViewBundleElement extends UmbLitElement {
+	@state()
+	_views: Array<ManifestCollectionView> = [];
+
+	@state()
+	_currentView?: ManifestCollectionView;
+
+	@state()
+	private _isOpen = false;
+
+	@state()
+	private _collectionRootPathname = '';
+
+	#collectionContext?: UmbCollectionContext<any, any>;
+
+	constructor() {
+		super();
+
+		this.consumeContext(UMB_COLLECTION_CONTEXT, (context) => {
+			this.#collectionContext = context;
+			if (!this.#collectionContext) return;
+			this._collectionRootPathname = this.#collectionContext.collectionRootPathname;
+			this.#observeViews();
+			this.#observeCurrentView();
+		});
+	}
+
+	#observeCurrentView() {
+		this.observe(this.#collectionContext!.currentView, (view) => {
+			this._currentView = view;
+		});
+	}
+
+	#observeViews() {
+		this.observe(this.#collectionContext!.views, (views) => {
```

Comment:
note about missing ControllerAlias, to make observe replace it self if this method is called again.